Cargo.toml

```diff
@@ -154,21 +154,21 @@ ascii = "1.1"
 bitflags = "2.4.2"
 bstr = "1"
 cfg-if = "1.0"
-chrono = "0.4.39"
+chrono = "0.4"
 criterion = { version = "0.3.5", features = ["html_reports"] }
 crossbeam-utils = "0.8.21"
 flame = "0.2.2"
 getrandom = { version = "0.3", features = ["std"] }
 glob = "0.3"
 hex = "0.4.3"
 indexmap = { version = "2.2.6", features = ["std"] }
-insta = "1.38.0"
+insta = "1.42"
 itertools = "0.14.0"
 is-macro = "0.3.7"
 junction = "1.2.0"
-libc = "0.2.169"
+libc = "0.2"
```

Comment:
we send a lot of patches to libc due to lack of definitions.
early 0.2 is not compatible.